### ISSUE TITLE
In the "class based components" lesson, the codebox provided has buttons with inaccessible color contrast.

### ISSUE THREAD

**Issue by eyaldavidszt** [TheOdinProject/curriculum#26366]
### Describe your suggestion

in this: https://www.theodinproject.com/lessons/node-path-react-new-class-based-components#assignment lesson's codebox, you can see the button color is black and the background color is dark blue. This creates a 2.44 contract ratio, falling below the minimal 4.5 for AA accessibility. 
https://webaim.org/resources/contrastchecker/

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-react-new-class-based-components

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [x] Would you like to work on this issue?

### (Optional) Discord Name

guzel

**Comment by eyaldavidszt** [comment]
I am opening to working on this issue, it would be simply to change the background color of the button, except I don't know how to work on issues and stuff 😅

**Comment by 01zulfi** [comment]
@eyaldavidszt thanks for opening the issue. The change will need to be made in our [react-examples repo](https://github.com/TheOdinProject/react-examples), more specifically the `class-component-example` folder in that repo. You can find instructions on how to setup the repo in the README. Let me know if you'd like to work on this issue. If not, that's fine as well.

**Comment by eyaldavidszt** [comment]
Yes I'll work on it!

**Comment by eyaldavidszt** [comment]
I did it, hope I did it right

**Comment by SaranshBangar** [comment]
@01zulfi Please assign me this issue, I can fix it

**Comment by Asartea** [comment]
There is already a fix for this at https://github.com/TheOdinProject/react-examples/pull/13

**Comment by 01zulfi** [comment]
closed with https://github.com/TheOdinProject/react-examples/pull/13
